.github/workflows/docker-image.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           images: |
             rob93c/stickerify
           tags: |
-            type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
This should remain as it is, since this is for the `latest` tag.

Reply:
Done